v03_pipeline/var/test/tasks/mock_complete_task.py

```diff
@@ -0,0 +1,6 @@
+import luigi
+
+
+class MockCompleteTask(luigi.Task):
```

Comment:
This probably belongs in `v03_pipeline/lib/test`.  `var` is meant for miscellaneous non-code files!